src/main/webapp/data/userMapData.json

```diff
@@ -1507,8 +1508,8 @@
       "John F. Kennedy University",
       "Johns Hopkins Bloomberg School of Public Health",
       "Johns Hopkins University",
+	  "Johnson County Community College",
```

Comment:
Honestly, this one could go either way, but since "&" is "and", it probably goes before Johnson *C*ounty